backend/src/AppServer.hs

```diff
@@ -185,6 +206,27 @@ reprocessReports = do
   forM_ reports processReport
   updateActiveStatus
 
+authGoogleLoginHandler :: IdToken -> AppM GoogleLoginResponse
+authGoogleLoginHandler idToken = do
+  httpConnMgr <- newManager
+  jwks <- liftIO (fetchGoogleJWKSet httpConnMgr) >>= either (\err -> throwError err401 {errBody = show err}) pure -- TODO cache this
```

Comment:
Uneducated/tentative nit, should an error on retrieving `jwks` be a 500 instead of a 401 since it happens prior to validating the user and it's retrieving public keys (I think)?

Reply:
Yeah, I think you're probably right. I'll change it :)